APE0.rst

```diff
@@ -0,0 +1,385 @@
+Members second the proposal.
+
+For Coordination Committee Member(s) to be removed, two-thirds of the active
+Voting Members must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Voting Member who has called
+for, or seconded such a vote, may not call for or second a vote to remove any
+other Coordination Committee member for one year from the original motion.
+
+The Ombudsperson
+----------------
+The Ombudsperson represents the interests of the Astropy community by providing
+an alternative point of contact for sensitive issues such as code of conduct
+violations and ethical concerns. Candidates for this project role are publicly
+nominated by the Coordination Committee, after which the Coordination Committee
+allows at least two weeks for comment, and then the nominee must be confirmed by
+two-thirds of the active Voting Members.
+
+The Ombudsperson has no term limit but can resign at any time, or be removed by
+the same process as being confirmed: the Coordination Committee initiates, there
+is a two-week comment period, and two-thirds of the active Voting Members have
+to approve the removal. In the period between removal/resignation and new
+appointment, the Coordination Committee will temporarily take over the
+responsibilities of the Ombudsperson, should the need arise.
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Responsibilities and Rights
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members may and should participate in formal votes on:
+
+* Adding new Voting Members
+* Removing Voting Members
+* Electing Coordination Committee members
+* Removing Coordination Committee members
+* Appointment or removal of the Ombudsperson
+* Changes to this Charter
+* Other matters for which the Coordination Committee believes a vote is
+  appropriate.
+
+The mechanism, timeline, and criteria for a decisive vote are specified in the
+respective sections of this document in the first six cases and by the
+Coordination Committee in the last case.
```

Comment:
technical suggestion: the bulleted list above should be a numbered list since they are referred to by number